src/pages/GamePage.js

```diff
@@ -81,6 +84,8 @@ function GamePage({ match }) {
 
   const [game, loadingGame] = useFirebaseRef(`games/${gameId}`);
   const [gameData, loadingGameData] = useFirebaseRef(`gameData/${gameId}`);
+  const [play_success] = useSound(foundSFX);
+  const [play_fail] = useSound(failSFX);
```

Comment:
Please use a consistent naming convention of `camelCase`. for `playSuccess`, `foundSfx`, etc.